2018 Robot/src/org/usfirst/frc/team4999/commands/lift/ZeroLift.java

```diff
@@ -32,13 +46,20 @@ protected void execute() {
 
     // Make this return true when this Command no longer needs to run execute()
     protected boolean isFinished() {
-        return RobotMap.liftZeroSwitch.get();
+    	if(pdp.getCurrent(RobotMap.liftMotor1PDP) > CUTOFF_CURRENT || pdp.getCurrent(RobotMap.liftMotor2PDP) > CUTOFF_CURRENT) {
```

Comment:
This overcurrent pattern shows up several times.  Is there any way to refactor it and reduce duplicate code?